### DISCUSSION TITLE
Error: canceling remote connection

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#20297] [Q&A]
minio log reports follow error intermittently:
![image](https://github.com/user-attachments/assets/6216fb8f-e8d8-4ea5-afb9-21db8e5919b9)
what's wrong with it, my network link is normal and minio server is healthy

my minio version is:
![image](https://github.com/user-attachments/assets/9447a000-2b60-49d4-b2f6-4fcee6aab616)

**Comment by klauspost** [comment]
Upgrade.
